nixos/modules/security/polkit.nix

```diff
@@ -58,6 +61,11 @@ in
 
     systemd.packages = [ pkgs.polkit.out ];
 
+    systemd.services.polkit.serviceConfig.ExecStart = [
+      ""
+      "${pkgs.polkit.out}/lib/polkit-1/polkitd ${optionalString (!cfg.debug) "--no-debug"}"
+    ];
```

Comment:
I originally considered a "minimally invasive" version:
```nix
    systemd.services.polkit.serviceConfig = mkIf cfg.debug {
      # Upstream unit has --no-debug
      ExecStart = [ "" "${pkgs.polkit.out}/lib/polkit-1/polkitd" ];
    };
```
But if upstream were to change the ExecStart in their shipped unit, our unit would become inconsistent depending on the value of `polkit.debug`.

Reply:
I'm confused by the current PR code. Isn't the flag inverted?

Reply:
I would say it's because there is only a "--no-debug" option and we have a debug option in the module so we have to invert the value of debug to determinate if we need "--no-debug". It's working locally tested if true debug messages show up in log, if false they don't (default currently).